src/types.rs

```diff
@@ -1120,6 +1246,36 @@ pub struct Policy<Attributes> {
     pub items:             Attributes,
 }
 
+#[cfg(feature = "std")]
+impl quickcheck::Arbitrary for OwnedPolicy {
+    //todo it is not clear to me if random instances of these are useful in tests
+    fn arbitrary(g: &mut Gen) -> OwnedPolicy {
+        OwnedPolicy {
+            identity_provider: quickcheck::Arbitrary::arbitrary(g),
+            created_at:        quickcheck::Arbitrary::arbitrary(g),
+            valid_to:          quickcheck::Arbitrary::arbitrary(g),
```

Comment:
It would be useful if valid_to >= created_at always.

Reply:
Done